src/test/java/com/google/api/codegen/testdata/csharp_gapic_client_library.baseline

```diff
@@ -3983,22 +3983,22 @@ namespace Google.Example.Library.V1
         /// <returns>
         /// A pageable asynchronous sequence of <see cref="string"/> resources.
         /// </returns>
-        public override IPagedAsyncEnumerable<FindRelatedBooksResponse, string> FindRelatedBooksAsync(
-            IEnumerable<string> names,
-            IEnumerable<string> shelves,
+        public override IPagedAsyncEnumerable<FindRelatedBooksResponse, BookName> FindRelatedBooksAsync(
```

Comment:
Was the decision for csharp to always iterate over the resource name in this case? In Java we still default to string, but the return object has an iterateAllAsBookName() method.